micrometer-core/src/main/java/io/micrometer/core/instrument/config/MeterFilter.java

```diff
@@ -45,14 +45,16 @@
     /**
      * Add common tags that are applied to every meter created afterward.
      *
+     * The common tags will not override tag values from a meter ID.
```

Comment:
They will also not override previously configured common tag MeterFilters that have the same tag key.

Reply:
@shakuzen Thanks for the feedback! I didn't include it as I thought the statement implied it. However, being explicit might be better. I added it in c971d65 as suggested.